ibm_catalog.json

```diff
@@ -154,6 +154,174 @@
+            ],
+            "short_description": "Creates and configures an IBM Cloud-Object-Storage instance and bucket",
+            "long_description": "This architecture supports creating an IBM Cloud-Object-Storage instance and bucket. You can also configure the following aspects of your instances: CBR rules, KMS encryption, Activity tracking and Monitoring",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/main/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/images/cos_icon.svg",
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "Create an IBM Cloud-Object-Storage instance",
+                    "description": "Create an IBM Cloud-Object-Storage instance."
+                },
+                {
+                    "title": "Create an IBM Cloud-Object-Storage bucket",
+                    "description": "Create either a cross-regional or regional Object Storage bucket."
+                },
+                {
+                    "title": "Create an IBM resource key",
+                    "description": "Create a resource key for the IBM Cloud-Object-Storage instance."
+                },
+                {
+                    "title": "Attach access tags",
+                    "description": "Attach access tags to the IBM Cloud-Object-Storage instance."
+                },
+                {
+                    "title": "Configure bucket retention",
+                    "description": "Configure retention policies at the IBM Cloud-Object-Storage bucket level."
+                },
+                {
+                    "title": "Enable object versioning",
+                    "description": "Supports enabling object versioning to keep multiple versions of an object in a bucket."
+                },
+                {
+                    "title": "Add archive policies",
+                    "description": "Add archive policies on the bucket."
+                },
+                {
+                    "title": "Record Activity Tracker events",
+                    "description": "Integrate Activity Tracker to track events."
+                },
+                {
+                    "title": "Monitor data",
+                    "description": "Integrate with the IBM Cloud-Monitoring service to monitor your Object Storage data."
+                },
+                {
+                    "title": "Encrypt data with key management services",
+                    "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
+                },
+                {
+                    "title": "Secure access with CBR",
+                    "description": "Create context-based restriction (CBR) rules for both instances and buckets."
+                }
+            ],
```

Comment:
Here, we should keep only main ones...like instance and bucket. Remaining instance specific and bucket specific features should be moved into corresponding flavors.